src/components/Notification/index.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import CloseIcon from './closeIcon';
+import ButtonIcon from './../ButtonIcon';
+import RenderIf from '../RenderIf';
+import Icon from './icons/index';
+import './styles.css';
+
+/**
+* Notifications serve as a confirmation mechanism & feedback that comes into the page at the top.
+*/
+export default function Notification(props) {
+    const {
+        className,
+        style,
+        icon,
+        title,
+        description,
+        onRequestClose,
+    } = props;
+
+    function getClassName() {
+        return classnames('rainbow-notification', className);
+    }
+
+    return (
+        <div className={getClassName()} style={style}>
+            <a className="rainbow-notification_anchor">
+                <RenderIf isTrue={!!icon}>
+                    <Icon icon={icon} />
+                </RenderIf>
+                <span>
+                    <RenderIf isTrue={!!title}>
+                        <h1 className="rainbow-notification_title">{title}</h1>
```

Comment:
here we should wrap this on a new component that check if title is a string:
```
export default function Header({ title }) {
    if (typeof title === 'string') {
        return (
            <h1 className="rainbow-notification_title">{title}</h1>
        );
    }
    return title;
}
```
We should leave that the user pass as title whatever he want and can be a node and if it is then it should not be wrapped by h1.